ui/app/AppLayouts/Chat/components/MessageContextMenu.qml

```diff
@@ -29,6 +29,11 @@ PopupMenu {
     property var emojiReactionsReactedByUser: []
     property var onClickEdit: function(){}
     property var reactionModel
+    property bool canPin: {
+        const nbPinnedMessages = chatsModel.messageView.pinnedMessagesList.count
```

Comment:
Will this work, considering this issue: https://github.com/status-im/status-desktop/issues/3014 ?

Reply:
I worked when I implemented it. I think the issue with the community bar is more specific, but good find